bin/parse-problem-doc.pl

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use experimental 'signatures';
+use feature 'say';
+
+my $pg_root;
+
+BEGIN {
+	use Mojo::File qw(curfile);
+	$pg_root = curfile->dirname->dirname;
+}
+
+use lib "$pg_root/lib";
+
+use Mojo::Template;
+use File::Basename qw(fileparse basename);
+use Getopt::Long;
+use File::Copy qw(copy);
+use YAML::XS qw(DumpFile);
+use Pod::Simple::Search;
+
+use SampleProblemParser qw(parseSampleProblem generateMetadata);
+
+my $problem_dir = "$pg_root/doc/sample-problems";
```

Comment:
This line needs to be updated for the new location.  I am not sure if you noticed that I added this.  This makes it so that you don't need to pass the `problem_dir` option if you are running this script on the default location.